src/design/themes.md

```diff
@@ -4,13 +4,13 @@ redirect_from:
   - /design/theme-responsive.html
 ---
 
-A theme is a collection of files that determines the visual presentation of your store. When you first install Magento, the design elements of the store are based on the “Default” Theme. In addition to the initial default theme that comes with your Magento installation, there is a wide variety of themes that are available “off the shelf” on Magento Connect.
+A theme is a collection of files that determines the visual presentation of your store. When you first install Magento, the design elements of the store are based on the _Default_ Theme. In addition to the initial default theme that comes with your Magento installation, there is a wide variety of available themes that you can use _as is_ or modify for your needs.
 
-A responsive theme adjusts the page layout to fit the view port of the device. Magento’s sample “Luma” theme has a flexible, responsive layout that can be viewed from the desktop, tablet, or mobile device.
+A responsive theme adjusts the page layout to fit the view port of the device. The sample _Luma_ theme has a flexible, responsive layout that can be viewed from the desktop, tablet, or mobile device.
 
-Magento themes include layout files, template files, translation files, and skins. A skin is a collection of supporting CSS, images, and JavaScript files that together, create the visual presentation and interactions that your customers experience when they visit your store. Themes and skins can be modified and customized by a developer or designer who has knowledge of Magento theme design and access to your server. To learn more, see the [Frontend Developer Guide][1].
+Magento themes include layout files, template files, translation files, and skins. A skin is a collection of supporting CSS, images, and JavaScript files that together, create the visual presentation and interactions that your customers experience when they visit your store. Themes and skins can be modified and customized by a developer or designer who has knowledge of Magento theme design and access to your server. To learn more, see our [Frontend Developer Guide][1].
```

Comment:
the link opens the guide at the top level.   the paragraph seems to be theme oriented.  consider using a link to the Theme topics.
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-overview.html

Reply:
will do